src/main/scala/org/apache/spark/sql/kinesis/KinesisSourceRDD.scala

```diff
@@ -111,6 +111,10 @@ private[kinesis] class KinesisSourceRDD(
     val recordPerRequest =
       sourceOptions.getOrElse("executor.maxRecordPerRead".toLowerCase(Locale.ROOT), "10000").toInt
 
+    val avoidTimeStampAsOffset =
+    sourceOptions.getOrElse("client.avoidTimeStampAsOffset"
+      .toLowerCase(Locale.ROOT), "false").toBoolean
```

Comment:
Do we want to avoid timestamp by default, given that we see issues?

Reply:
This is a fair point. Will remove the flag to avoid timeStamp.